src/editor.js

```diff
@@ -1028,7 +1007,7 @@ function saveAsDialog() {
         });
 
         // Until release to production, make sure we are on demo before displaying the propc option
-        if (inDemo === 'demo') {
+        if (isExperimental.indexOf('saveas') > -1) {
             $("#save-as-board-type").append($('<option />').val('propcfile').text('Propeller C (code-only)'));
```

Comment:
Save as feature - allowed a project to be saved as a different board type.  Need to revisit how this would look in solo.

Reply:
We have been discussing this very detail internally. Various blocks do not make sense when coupled with specific board types. Other blocks may peacefully co-exist in a select group of board types, but we have to be concerned with details, such a I/O pin numbering and availability. The blocks not in one of the above categories are universal and can be used in any project.

One though was to include metadata in each block that would declare which of the above groupings apply to the block. Other logic would then be able to determine if the block can be used with a specific board type or can be tolerated with a specific board type. A gross example of this would be trying to import a S3 project into an existing Flip project. Not much overlap there. But importing a Flip project into an Activity board project might actually work without any modifications.

Once we have a better idea of exactly what we're looking at, I will open a new issue to explore it further.

Reply:
Hangouts me on this one - we have some control already in place to address what you are talking about.  I'll show you where it's at and what it does.